### DISCUSSION TITLE
Trying to set GTK theme

### DISCUSSION THREAD

**Discussion by cyberfishdev** [sioodmy/dotfiles#69] [Q&A]
Hi sioodmy, first of all thank you for sharing you great work with the community. 
I know this config is only for your private use, but I am also trying to get a similar result of Hyprland in nixos. 
I'm trying in vain, to set the gtk themes system wide with home manager and therefore did the same steps you did. But no matter what I configured, the gtk / icon / cursor theme never gets applied. Are there any hidden steps in the hyprland / wayland / xdg or dconf config I have to take, that the theme gets applied. 
What steps did you have to do ( instead of writing own pkgs for the themes ), so they could be applied ? 
I basically used the same gtk.nix as you did (attached here: [gtk.md](https://github.com/sioodmy/dotfiles/files/9995025/gtk.md)) in your config, but it might not be enough to get the themes applied.
Thank you very much in advance ;)

**Comment by sioodmy** [comment]
https://github.com/sioodmy/dotfiles/blob/d51c464f99b55a74371dc7f9d2224f1c5dd2d988/system/default.nix#L10-L21
